syntax = "proto3";
option optimize_for = LITE_RUNTIME;


message skeletonData_proto
{
	repeated uint32 jointIndicies = 1;
	repeated uint32 jointParentIndicies = 2;
	repeated string jointNames = 3;
}
